tests/test_wps_generate_climos.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from pywps import Service
+from pywps.tests import assert_response_success
+
+from .common import client_for, TESTDATA
+from thunderbird.processes.wps_generate_climos import GenerateClimos
+import owslib.wps
+
+
+@pytest.mark.online
+@pytest.mark.parametrize(
+    ("opendap"), [(TESTDATA["test_opendap"])],
+)
+@pytest.mark.parametrize(
+    (
+        "operation",
+        "climo",
+        "resolutions",
+        "convert_longitudes",
+        "split_vars",
+        "split_intervals",
+        "dry_run",
+    ),
+    [
+        ("mean", "6190", "yearly", "True", "True", "True", "False"),
+        ("std", "7100", "seasonal", "False", "False", "False", "True"),
```

Comment:
Should probably have a test case for `resolutions="all"` here. And bonus... `resolutions={'all', 'yearly'}` in which case, I don't think your code will do the right thing :wink: